core/chains/cosmos/chain_set.go

```diff
@@ -27,7 +32,7 @@ var (
 
 // ChainSetOpts holds options for configuring a ChainSet.
 type ChainSetOpts struct {
-	Config           coreconfig.AppConfig
+	Config           pg.QConfig
```

Comment:
/nit rename? Not clear to me what QConfig is about

Reply:
I think this is more of a package/type name problem, but that's probably out of scope due to noise:
```suggestion
	Config           postgres.QueryConfig
```